src/views/components/avatar/common/avatarLibraryGridView.tsx

```diff
@@ -16,8 +17,10 @@ export function AvatarLibraryGridView({
     require('./styles.css');
   }, []);
 
+  const darkMode = useContext(DarkModeContext);
+
   return (
-    <div className="grid-container">
+    <div className={`grid-container ${darkMode ? 'dark' : ''}`}>
```

Comment:
Let's import dark.css into the root React guy, and then shove our css specialties there.